### PR TITLE
capability: fix return type for property, command

### PR DESCRIPTION
**Pull request by webispy**
Change the current return type 'void' to 'bool' to check result for
getProperty(), getProperties(), receiveCommand() and sendCommand() APIs.

Signed-off-by: Inho Oh <inho.oh@sk.com>

### CODE REVIEW

File: src/capability/audio_player_agent.cc [hunk 1]
```diff
@@ -62,8 +62,9 @@ void AudioPlayerAgent::initialize()
     }
 
     std::string volume_str;
-    capa_helper->getCapabilityProperty("Speaker", "music", volume_str);
-    volume = std::stoi(volume_str);
+    if (capa_helper->getCapabilityProperty("Speaker", "music", volume_str)) {
+        volume = std::stoi(volume_str);
+    }
```
Comment (kimhyungrok) [inline src/capability/audio_player_agent.cc:68]: How about to remove brace?

File: src/capability/tts_agent.cc [hunk 2]
```diff
@@ -64,8 +64,9 @@ void TTSAgent::initialize()
     }
 
     std::string volume_str;
-    capa_helper->getCapabilityProperty("Speaker", "voice_command", volume_str);
-    volume = std::stoi(volume_str);
+    if (capa_helper->getCapabilityProperty("Speaker", "voice_command", volume_str)) {
+        volume = std::stoi(volume_str);
+    }
```
Comment (kimhyungrok) [inline src/capability/tts_agent.cc:69]: How about to remove brace?

File: src/capability/audio_player_agent.cc (hunk 1 above)
Comment (webispy) [reply src/capability/audio_player_agent.cc:67]: Thank you. Your suggestion looks better.

File: src/capability/tts_agent.cc (hunk 2 above)
Comment (webispy) [reply src/capability/tts_agent.cc]: Your suggestion looks better.

**Pull request merged**